descriptors/ModuleDescriptor-template.json

```diff
@@ -174,6 +174,16 @@
           ],
           "modulePermissions": []
         },
+        {
+          "methods": [
+            "DELETE"
+          ],
+          "pathPattern": "/consortia/{consortiumId}/publications/{publicationId}/results",
```

Comment:
shouldn't it be '/consortia/{consortiumId}/publications/{publicationId} ?'

Reply:
Thank you, corrected.